File: GestionEventosDeportivosBackend/Models/ParticipanteModel.cs
using System.ComponentModel.DataAnnotations;

namespace GestionEventosDeportivos.Models
{
    public class ParticipanteModel
    {
        [Key]
        public int participante_id { get; set; }

        [Required]
        [StringLength(50)]
        public string nombre { get; set; }

        [Required]
        [StringLength(50)]
        public string apellido { get; set; }

        [Required]
        [EmailAddress]
        [StringLength(100)]
        public string email { get; set; }

        [StringLength(15)]
        public string telefono { get; set; }

        // Relación con inscripciones
        public virtual ICollection<InscripcionModel> Inscripciones { get; set; } = new List<InscripcionModel>();
    }
}

File: GestionEventosDeportivosFrontend/src/app/models/participante.model.ts
export interface Participante {
  participante_id: number;
  nombre: string;
  apellido: string;
  email: string;
  telefono: string;
  nombre_completo: string;
}

export interface CreateParticipante {
  nombre: string;
  apellido: string;
  email: string;
  telefono: string;
}

export interface UpdateParticipante {
  nombre: string;
  apellido: string;
  email: string;
  telefono: string;
}